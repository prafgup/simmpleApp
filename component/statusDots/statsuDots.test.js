import React from "react";
import {shallow} from "enzyme";

import StatusDots from "./index";


describe("Status Dots",()=>{

    describe("Status Dots",()=>{
        let component ;

        beforeEach(()=>{
            component = shallow(<StatusDots/>);
        });

        it("Should match to snapshot",()=>{
            expect(component).toMatchSnapshot("Status Dots");
        });

        it("Should Render all 3 Dots",()=>{
            expect(component.find("[data-test='dots']").length).toBe(3);
        });
    })
});