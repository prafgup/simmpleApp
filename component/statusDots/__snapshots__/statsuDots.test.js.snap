// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status Dots Status Dots Should match to snapshot: Status Dots 1`] = `ShallowWrapper {}`;
